.github/workflows/atm-check-queries-run.yml

```diff
@@ -1,13 +1,80 @@
 name: ATM Check Queries Run
 
-# This check is required, therefore we must run it on all PRs, even if only Markdown has changed.
+env:
+  AZURE_STORAGE_URL: "https://atmcodeqldata.blob.core.windows.net"
+  DB_NAME: "AmanSultanBaig/SignIn-SignUp-System-with-Nodejs"
```

Comment:
> It's good to avoid making our PR checks depend on repos that we don't control

Well yes I agree and this is exactly the problem I brought up and I thought we had agreed on this method, but I must have misunderstood.

> I would suggest we instead copy the small test project

I'm not sure I followed the suggestion, do you mind pushing a commit?

Reply:
If I add a commit, I won't be able to review this PR any more due to the new branch protection rules :)

I was thinking we could check the files under [this directory](https://github.com/github/codeql-action/tree/main/tests/ml-powered-queries-repo) into this repo under a new `config/atm` directory (within https://github.com/github/codeql/tree/main/config).  Then when creating the database we can run `codeql database create --source-root config/atm`.